packages/datadog-plugin-prisma/src/client.js

```diff
@@ -0,0 +1,61 @@
+'use strict'
+const DatabasePlugin = require('../../dd-trace/src/plugins/database')
+class PrismaClientPlugin extends DatabasePlugin {
+  static get id () { return 'prisma' }
+  static get operation () { return 'client' }
+  static get system () { return 'prisma' }
+  static get prefix () {
+    return 'tracing:apm:prisma:client'
+  }
+
+  bindStart (ctx) {
+    const service = this.serviceName({ pluginConfig: this.config })
+    const resource = formatResourceName(ctx.resourceName, ctx.attributes)
+
+    const options = { service, resource }
+
+    if (ctx.resourceName === 'operation') {
+      options.meta = {
+        prisma: {
+          method: ctx.attributes.method,
+          model: ctx.attributes.model,
+          type: 'client'
+        }
+      }
+    }
+    const operationName = this.operationName({ operation: this.operation })
+    this.startSpan(operationName, options, ctx)
+
+    return ctx.currentStore
+  }
+
+  end (ctx) {
+    // Only synchronous operations would have `result` on `end`.
+    if (!ctx.hasOwnProperty('result')) return
```

Comment:
Please do not use `.hasOwnProperty()`. Instead, please always use

```suggestion
    if (!Object.hasOwn(ctx, 'result')) return
```